h5py/h5g.pyx

```diff
@@ -406,7 +406,7 @@ cdef class GroupID(ObjectID):
         Retrieve a copy of the group creation property list used to
         create this group.
         """
-        return PropGCID(H5Gget_create_plist(self.id))
```

Comment:
Why the change from PropGCID to propwrap?

Reply:
because I found that the `get_create_plist` in h5d, h5f, h5t do the same thing. 😃

Reply:
That's reasonable enough. For future reference, it's a good idea to keep pull requests focused, so a documentation PR doesn't change code at all (except code in docs & examples). But this instance is OK.